utils/certs/gen_certificates.sh

```diff
@@ -115,10 +115,10 @@ extendedKeyUsage = clientAuth
 function generate_ca_cert () {
     echo "Generating Private CA Root Certificate"
     # Generate Private key and set permissions
-    openssl genrsa -out "${PRIVATE}/daosCA.key" 4096
+    openssl genrsa -out "${PRIVATE}/daosCA.key" 3072
```

Comment:
Why the drop in key size?

Reply:
To be honest I'm not entirely sure. This was the guidance from the crypto team. I'd imagine its due to the fact that 3072 bit keys are computationally difficult enough to break and since its software not hardware that we can update the number if that turns out not to be true in the future.

Reply:
I think probably a balance between performance and security. 
This is what I got on boro-59, there is no option for rsa 3072 size, but it shows the CPU time for the key size:
[boro-59:/home/huakuang]$ openssl speed rsa2048 rsa4096
                               sign    verify        sign/s verify/s
rsa 2048 bits 0.000791s 0.000033s   1264.8  30152.7
rsa 4096 bits 0.007899s 0.000121s    126.6   8276.1